README.md

```diff
@@ -951,10 +951,26 @@ They can be overridden for testing purposes in [the same way as other settings](
 
 ##### AWS-specific settings
 
-AWS-specific settings are automatically set based on calls to the Instance MetaData Service (IMDS).
+* `settings.aws.config`: The base64 encoded content to use for AWS configuration (e.g. `base64 ~/.aws/config`).
+* `settings.aws.credentials`: The base64 encoded content to use for AWS credentials (e.g. `base64 ~/.aws/credentials`).
+* `settings.aws.profile`: The profile name to use from the provided `config` and `credentials` settings.
+
+  For example:
+
+  ```toml
+  [settings.aws]
+  profile = "myprofile"
+  ```
+
+  **Note**: If `settings.aws.profile` is not set, it will fallback to using the "default" profile.
+
+  **Note:** The `config`, `credentials`, and `profile` are optional and do not need to be set when using an Instance Profile when running on an AWS instance.
 
 * `settings.aws.region`: This is set to the AWS region in which the instance is running, for example `us-west-2`.
 
+  The `region` setting is automatically inferred based on calls to the Instance MetaData Service (IMDS).
```

Comment:
This is only true in AWS variants, right? In VMWare / metal variants, IMDS isn't used to automatically fetch the region. Should this be specified in the note?

Reply:
Added some more wording.